src/Symfony/Component/Notifier/Bridge/FakeSms/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/fake-sms-notifier",
+    "type": "symfony-bridge",
+    "description": "Fake SMS (as email during development) Notifier Bridge.",
+    "keywords": ["sms", "development", "email", "notifier", "symfony"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "James Hemery",
+            "homepage": "https://github.com/JamesHemery"
+        },
+        {
+            "name": "Yield Studio",
+            "homepage": "https://github.com/YieldStudio"
+        },
```

Comment:
This looks more like an advertisement than an author. 
```suggestion
```

Reply:
This is from the original package, but no worries.